specification/adp/resource-manager/readme.md

```diff
@@ -28,18 +28,21 @@ These are the global settings for the adp.
 title: AdpManagementClient
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2020-07-01-preview
+tag: 2021-02-01-preview
 ```
 
-### Tag: package-2020-07-01-preview
-
-These settings apply only when `--tag=package-2020-07-01-preview` is specified on the command line.
-
-```yaml $(tag) == 'package-2020-07-01-preview'
+```yaml $(tag) == '2020-07-01-preview'
```

Comment:
usually we have a tag with the prefix of `package` therefore could we still keep this convention?